tests/autobahn/overrides.yaml

```diff
@@ -0,0 +1,25 @@
+---
+overrides:
+  search-module:
+    shortdef: 'Navigation.Overriden(fully-overriden: false)'
+  root:
+    shortdef: 'Controller.Overriden(added: true)'
+  root.window.contents.1:
+    type: Layout.Overriden
+  root.window.search:
+    properties:
+      fully-overriden: true
```

Comment:
I was a bit confused by this at first, as I thought you were introducing a new property that you had to set if you wanted to fully override a module. Maybe it should be called something like `override-sample-property`?

Reply:
The meaning behind `fully` name is more subtle than I expected then, notice that :
1) the search node references a variable.
2) but the variable is overridden, in the overrides section.
3) but the overridden version has that property set to false, because
4) we also have a custom override for that property.

the reason for this is to include in the test the order in which these are processed.

1) variables replacement and overridden, and then
2) custom overrides applied on top of the final result.

Reply:
Well, maybe still put `example` or `sample` in there? I was confused whether it was part of the overriding API or not.